core/storage/question/gae_models.py

```diff
@@ -262,6 +262,23 @@ def create(cls, question_id, skill_id, skill_difficulty):
         )
         return question_skill_link_model_instance
 
+    @classmethod
+    def get_total_question_count_for_skill_ids(cls, skill_ids):
+        """Returns the number of questions assigned to the given skill_ids.
+
+        Args:
+            skill_ids: list(str). Skill IDs for which the question count is
+                requested.
+
+        Returns:
+            question_count: int. The number of questions assigned to the
```

Comment:
I think I had raised this comment before i.e, we don't need the variable name in the return's docstring, right?

Reply:
Done.